app/templates/views/guidance/edit-and-format-messages.html

```diff
@@ -96,4 +97,20 @@ <h2 class="heading-medium" id="optional-content">Add optional content</h2>
 
   <p class="govuk-body">If you upload a list, the column names need to match the optional content in your template.</p>
 
+  <p class="govuk-body">You cannot use personalisation within optional content.</p>
```

Comment:
```suggestion
  <p class="govuk-body">You cannot use placeholders within optional content.</p>
```

This is more specific to queries we've had in the past.